config/gulp/gulp.browser-sync.js

```diff
@@ -27,8 +27,8 @@ gulp.task('browser-sync', function() {
       ]
     },
     files: [
-      `${pathsHelper.tmp}/*.css`,
-      `${pathsHelper.tmp}/*.html`
```

Comment:
Wenn das draussen ist, wird aber das index.html nicht mehr von browser-sync angeschaut oder?

Reply:
true dat, included it again, thanks!